src/http.cc

```diff
@@ -1393,6 +1395,16 @@ HttpStateData::truncateVirginBody()
     }
 }
 
+/// called when an incomplete reply body preliminary ends with unexpected EOF
```

Comment:
Nitpick: "unexpected" kind of implies "incomplete" and "preliminary" here.

```suggestion
/// called on a premature EOF discovered when reading response body
```

Reply:
Done.